### PR TITLE
Auto wrap single nodes in snippets

### PR DESCRIPTION
**Pull request by cbarrete**
Last one of those guys :)

### CODE REVIEW

**Comment by L3MON4D3** [comment]
Ahh, that's a shame :D

File: lua/luasnip/nodes/snippet.lua [hunk 1]
```diff
@@ -44,6 +44,13 @@ local function S(context, nodes, condition, ...)
 			return true
 		end
 	end
+
+	local nodes = nodes
+	if not nodes[1] then
+		-- Then it's a node-table rather than an array of nodes: auto-wrap it
+		nodes = { nodes }
+	end
```
Comment (L3MON4D3) [inline lua/luasnip/nodes/snippet.lua:52]: Very minor, but I think checking for `type(nodes) == "table"` is a bit more readable.

Comment (cbarrete) [reply lua/luasnip/nodes/snippet.lua:52]: That's the point, I can't. Both nodes *and* node lists are tables, so that doesn't help (hence the comment).

Comment (L3MON4D3) [reply lua/luasnip/nodes/snippet.lua:52]: Ohh yeah, you're right, mb!

**Pull request merged**